src/img/geltungsbereicheLegends/index.js

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable import/no-duplicates */
```

Comment:
@oterral here we still need to wait for the client to approve the legend, so we duplicate GA German A3 and A4 for all for now